clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -37,7 +37,15 @@ public RemoveMembersFromConsumerGroupOptions(Collection<MemberToRemove> members)
         this.members = new HashSet<>(members);
     }
 
+    public RemoveMembersFromConsumerGroupOptions() {
+        this.members = new HashSet<>();
```

Comment:
Collections.emptySet() makes more sense since it is immutable.

Reply:
Make sense, fixed